apps/server/src/modules/school/service/school-year.service.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { SchoolYearRepo } from '@shared/repo/schoolyear';
+import { Test, TestingModule } from '@nestjs/testing';
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { setupEntities } from '@shared/testing';
+import { schoolYearFactory } from '@shared/testing/factory/schoolyear.factory';
+import { SchoolYear } from '@shared/domain';
+import { SchoolYearService } from './school-year.service';
+
+describe('SchoolYearService', () => {
+	let module: TestingModule;
+	let service: SchoolYearService;
+
+	let schoolYearRepo: DeepMocked<SchoolYearRepo>;
+
+	beforeAll(async () => {
+		module = await Test.createTestingModule({
+			providers: [
+				SchoolYearService,
+				{
+					provide: SchoolYearRepo,
+					useValue: createMock<SchoolYearRepo>(),
+				},
+			],
+		}).compile();
+
+		service = module.get(SchoolYearService);
+		schoolYearRepo = module.get(SchoolYearRepo);
+
+		await setupEntities();
+	});
+
+	afterAll(async () => {
+		await module.close();
+	});
+
+	describe('getCurrentSchoolYear', () => {
+		const setup = () => {
```

Comment:
Please add the when describe 
Second describe when a error is occured  ..it should be pass this without catch or what ever.

Reply:
This comment is marked as solved, but nothing is changed.